.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+version = 3.8.1
+runner.dialect = scala213
```

Comment:
We should probably use `scala213source3`: https://scalameta.org/scalafmt/docs/configuration.html#scala-2-with--xsource3